docs/sql_reference/src/asciidoc/_chapters/sql_utilities.adoc

```diff
@@ -48,6 +49,326 @@ to connect to and issue SQL utilities, see the
 {docs-url}/client_installation/index.html[_{project-name} Client Installation Guide_].
 
 <<<
+[[cleanup_statement]]
+== CLEANUP Statement
+
+The CLEANUP Statement cleans up objects whose information is inconsistent in the metadata. This command can be used to clean up individual objects or all objects in a schema. This command can also be used to locate all inconsistent objects in the Trafodion metadata and then clean them up.
+
+Inconsistent objects can be cleaned up and removed from the metadata and HBase by using one of the following options.
+
+```
+CLEANUP {obj-type} {obj-name} [ ,UID {object-uid} ]

```

Comment:
Thanks Dave. :)
But I'm afraid `RETURN DETAILS` is not supported here.
Ming helped me perform a test using the sqlci, it returned a syntax error, then we checked the parser/sqlparser.y and found that `RETURN DETAILS is not yet supported` , details are as below (circled in red):
![return details is not yet suported](https://cloud.githubusercontent.com/assets/20532956/25522260/d5e8d2b6-2c34-11e7-98ed-47999b765936.PNG)